spec/models/user_spec.rb

```diff
@@ -941,4 +941,11 @@ def user_from_authorization_service(service_name, signed_in_resource, cta_varian
       expect(user.receives_follower_email_notifications?).to be(true)
     end
   end
+
+  describe "#email_daily_digest_notifications column exists" do
+    it "returns false by defult for daily digest notifications" do
+      user.assign_attributes(email_daily_digest_notifications: false)
```

Comment:
good testing to make sure this attribute defaults as false when user is created.